modules/caddyhttp/reverseproxy/fastcgi/caddyfile.go

```diff
@@ -303,6 +309,14 @@ func parsePHPFastCGI(h httpcaddyfile.Helper) ([]httpcaddyfile.ConfigValue, error
 				fcgiTransport.WriteTimeout = caddy.Duration(dur)
 				dispenser.Delete()
 				dispenser.Delete()
+
+			case "capture_stderr":
+				args := dispenser.RemainingArgs()
+				dispenser.Delete()
+				for range args {
+					dispenser.Delete()
+				}
+				fcgiTransport.CaptureStderr = true
```

Comment:
Maybe `if dispenser.NextArg() { return nil, dispenser.ArgErr() }` would be a better way to handle this, since no args are expected.

Reply:
Kinda contradicts what I said to do btw; This matches the code we have for `resolve_root_symlink` which also has no options. Either one is fine though, as long as we're consistent.

Reply:
Good point, but yeah... I think I prefer the simpler form if it works. I'm not sure why reading all the arguments and ignoring them is helpful.

Reply:
Moreso abundance of caution with deleting the tokens, but throwing an error is technically more valid since we don't support args for those (yet). But extremely unlikely anyone will encounter this as a problem either way.